CI failed

```
Run nix-instantiate --eval -E '(import <nixpkgs> {}).lib.version'
  nix-instantiate --eval -E '(import <nixpkgs> {}).lib.version'
  shell: /usr/bin/bash -e {0}
  env:
    NIX_PATH: nixpkgs=channel:nixos-unstable
error: could not set permissions on '/nix/var/nix/profiles/per-user' to 7[5](https://github.com/natsukium/nur-packages/actions/runs/4309835613/jobs/7517666451#step:4:5)5: Operation not permitted
Error: Process completed with exit code 1.
```

https://github.com/natsukium/nur-packages/actions/runs/4309835613

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -48,9 +48,6 @@ jobs:
       uses: cachix/install-nix-action@v19
       with:
         nix_path: "${{ matrix.nixPath }}"
-        # nix 2.6 breaks restrict-eval, when using the NIX_PATH
-        # see https://github.com/NixOS/nix/issues/5980
-        install_url: https://releases.nixos.org/nix/nix-2.5.1/install
         extra_nix_config: |
           experimental-features = nix-command flakes
           access-tokens = github.com=${{ secrets.GITHUB_TOKEN }}

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -45,7 +45,7 @@ jobs:
     - name: Checkout repository
       uses: actions/checkout@v3
     - name: Install nix
-      uses: cachix/install-nix-action@v19
+      uses: cachix/install-nix-action@v20
       with:
         nix_path: "${{ matrix.nixPath }}"
         extra_nix_config: |

